server/athenian/api/models/state/models.py

```diff
@@ -67,3 +68,24 @@ def count_items(ctx):
                          onupdate=count_items)
 
     count_items = staticmethod(count_items)
+
+
+class UserAccount(Base):
+    """User<>account many-to-many relations."""
+
+    __tablename__ = "user_accounts"
+
+    user_id = Column("user_id", String(256), primary_key=True)
+    account_id = Column("account_id", Integer(), ForeignKey("accounts.id", name="fk_user_account"),
```

Comment:
I'd add a unique constraint on the tuple `(user_id, account_id)`.

Reply:
They are both `primary_key=True` so according to the docs they should be unique together (composite primary key).